### PR TITLE
User Story #12 - Home Page Created

### PR DESCRIPTION
**Pull request by vikadia7x**
Pull request to merge the home page into develop branch

- Home page created which renders movie showtimes according to user location
- User has the ability to change the zip and get movie showtimes of his choice

- The folder structure remains constant

### CODE REVIEW

File: myProject/MovieShowTimeFinder/showtimefinder/templates/home.html [hunk 1]
```diff
@@ -1,20 +1,77 @@
...
+
+<p style="text-align:right;">Logout</p>
+<p style="text-align:right;">Edit Profile</p>
+
+
+<h3 align="center">Welcome User! Your homepage gives you a list of movies in theatres near you.!!</h3>
+
+<h4 align="center" style="color: red;">Enter the Zip Code</h4>
+
+<style>
```
Comment (ShivamVerma920302) [inline myProject/MovieShowTimeFinder/showtimefinder/templates/home.html]: Move Styling to a static CSS file

File: myProject/MovieShowTimeFinder/showtimefinder/views.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def home(request):
             'form': form,
             'movielist' : movielist
         }
-        #print(args['movielist'][0][0])
+        print(args['movielist'])
```
Comment (ShivamVerma920302) [inline myProject/MovieShowTimeFinder/showtimefinder/views.py]: Remove the print statement

File: myProject/MovieShowTimeFinder/showtimefinder/templates/home.html (hunk 1 above)
Comment (vikadia7x) [reply myProject/MovieShowTimeFinder/showtimefinder/templates/home.html]: Will create a consolidated CSS in the develop branch.

File: myProject/MovieShowTimeFinder/showtimefinder/views.py (hunk 2 above)
Comment (vikadia7x) [reply myProject/MovieShowTimeFinder/showtimefinder/views.py]: Comment Resolved.

**Pull request merged**